Kudu.Console/Program.cs

```diff
@@ -160,7 +160,7 @@ private static int Main(string[] args)
             {
                 try
                 {
-                    deploymentManager.DeployAsync(gitRepository, changeSet: null, deployer: deployer, clean: false)
+                    deploymentManager.DeployAsync(gitRepository, changeSet: null, deployer: deployer, clean: false, needFileUpdate: true, swapAfterDeploy: true)
```

Comment:
No clear looking at the code why we always swap after deploy. What if auto-swap is not enabled? Or maybe the flag doesn't mean what it sounds like?